README.md

```diff
@@ -30,8 +30,8 @@ originally developed by Nestybox ([acquired by Docker][docker-acquisition] on
 *   **Enables containers to run same workloads as VMs**:
 
     *   With Sysbox, containers can run system-level software such as systemd,
-        Docker, Kubernetes, K3s, buildx, legacy apps, and more seamlessly &
-        securely.
+        Docker, Kubernetes, K3s, buildx (including multi-arch builds), legacy
```

Comment:
One thing is to build a multi-arch image, and another is to be able to run it; one thing doesn't necessarily involve the other one. You're doing both with this work, so I'd highlight it. Example:

``` 
With Sysbox, containers can run system-level software such as systemd, Docker, Kubernetes, K3s, buildx (multi-arch support), legacy apps, multi-arch apps, and more seamlessly & securely.
```

As a next step around the "run multi-arch apps" feature, we could automatically detect when an image with an arch != local is about to run and have `sysbox-runc` automatically launch `multiarch/qemu-user-static` to support it.

Reply:
Thanks, I added your first suggestion. 

> As a next step around the "run multi-arch apps" feature, we could automatically detect when an image with an arch != local is about to run and have sysbox-runc automatically launch multiarch/qemu-user-static to support it.

Yes something to think about.